packages/service/src/service/base.ts

```diff
@@ -97,7 +97,7 @@ export class BtcAssetsApiBase implements BaseApis {
       throw BtcAssetsApiError.withComment(ErrorCodes.ASSETS_API_RESPONSE_ERROR, comment, context);
     }
     if (status !== 200) {
-      return void 0 as T;
+      return undefined as T;
```

Comment:
Same as above

Reply:
This part is expected but the if statement was not clear, I'll update it in another PR.